### PR TITLE
Be more clear about add to app

### PR DESCRIPTION
**Pull request by blasten**
https://github.com/flutter/flutter/issues/44050#issuecomment-553021066

### CODE REVIEW

File: src/docs/development/androidx-migration.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ To create a new project run:
 flutter create --androidx -t <project-type> <new-project-path>
 ```
 
-### How do I migrate my existing module project to AndroidX?
+### Add to App
```
Comment (jmagman) [inline src/docs/development/androidx-migration.md]: @xster @RedBrogdon Do we have a real marketing term for this?  We have "add-to-app", "existing app", "module".  "Module" is an overloaded term for nearly every platform and language.

File: src/docs/development/androidx-migration.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ steps in previous section.
 
 * **Apps and Modules**
```
Comment (jmagman) [inline src/docs/development/androidx-migration.md]: This still references Modules.

Comment (blasten) [reply src/docs/development/androidx-migration.md]: The project is still a module, but it's used in the context of add to app.

File: src/docs/development/androidx-migration.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ To create a new project run:
 flutter create --androidx -t <project-type> <new-project-path>
 ```
 
-### How do I migrate my existing module project to AndroidX?
+### Add to App
 
 If your project is a module and contains a `.android` directory, add the following
```
Comment (xster) [inline src/docs/development/androidx-migration.md]: If your Flutter project is a module type for adding to an existing Android app, and contains a ...

**Comment by xster** [comment]
LGTM

File: src/docs/development/androidx-migration.md (hunk 1 above)
Comment (blasten) [reply src/docs/development/androidx-migration.md]: resolved offline

**Pull request merged**